modules/react/layout/lib/utils/mergeStyles.ts

```diff
@@ -40,71 +39,23 @@ function isStyleProps(prop: string): boolean {
 }
 
 /**
- * This is an opinionated function that will force style merging with Emotion's runtime APIs,
- * including [styled components](https://emotion.sh/docs/styled) and the [css
- * prop](https://emotion.sh/docs/css-prop). It will also handle style props for as long as those are
- * supported. `mergeStyles` is exported from the layout package since it supports style props
- * defined in the layout packages.
- *
- * It works by detecting styles added via Emotion's runtime APIs and forcing styling merging if use
- * of runtime APIs have been detected. If only `createStyles` were used to style a component, the
- * faster non-runtime styling will be used.
- *
- * You can use `mergeStyles` if you wish to use {@link createStyles} on your own components and want
- * your components to be compatible with Emotion's runtime styling APIs.
- *
- * ```tsx
- * import {createStyles, CSProps} from '@workday/canvas-kit-styling';
- * import {mergeStyles} from '@workday/canvas-kit-react/layout';
- *
- * interface MyComponentProps extends CSProps {
- *   // other props
- * }
- *
- * const myStyles = createStyles({
- *   background: 'green',
- *   height: 40,
- *   width: 40
- * })
- *
- * const MyComponent = ({children, ...elemProps}: MyComponentProps) => {
- *   return (
- *     <div
- *       {...mergeStyles(elemProps, [myStyles])}
- *     >
- *       {children}
- *     </div>
- *   )
- * }
- *
- * const StyledMyComponent(MyComponent)({
- *   background: 'red'
- * })
- *
- * const myOverridingStyles = createStyles({
- *   background: 'blue'
- * })
- *
- * // now everything works. Without `mergeStyles`, the last component would be a red box
- * export default () => (
- *   <>
- *     <MyComponent>Green box</MyComponent>
- *     <StyledMyComponent>Red box</StyledMyComponent>
- *     <StyledMyComponent cs={myOverridingStyles}>Blue box</StyledMyComponent>
- *   </>
- * )
- * ```
- *
+ * This function has the same signature as {@link handleCsProp} and also calls `handleCsProps`, but
+ * adds support for style props. It can be used as a drop-in replacement for `handleCsProps`.
```

Comment:
I've updated this JSDoc to decrease the significance of `mergeStyles` and promote `handleCsProp`. The JSDoc was moved to `handleCsProp`